core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -3670,6 +3678,12 @@ private static RelDataType deriveTypeMapFromEntries(SqlOperatorBinding opBinding
           OperandTypes.STRING_STRING,
           SqlFunctionCategory.SYSTEM);
 
+  @LibraryOperator(libraries = {BIG_QUERY})
+  public static final SqlFunction JSON_EXTRACT =
+      new SqlFunction("JSON_EXTRACT", SqlKind.OTHER_FUNCTION,
+          ReturnTypes.VARCHAR_2000_NULLABLE, null,
```

Comment:
Same as above, please verify if null is allowed for BQ. If not, we should make it VARCHAR_2000

Reply:
Yeah sure @shirin-hasan I have been verified .NULL operand is allowed in BQ.